### DISCUSSION TITLE
Some experimental thougts...

### DISCUSSION THREAD

**Discussion by flowerhorse** [brunoherbelin/vimix#77] [Ideas]
While working with vimix prepairing my Exhibition I've also worked with 'generated media' in form of white and black rectangles. I've used them for light/darking  the projection area also for adjusting brightness and contrast. 
That it succeeded was rather a coincidence and remained limited in its usefulness.

One could use this possibility however also purposefully:

One could use the 'generated' pads like a movable filter, in order to partially influence the video image. The change would then not be on 'white' or black, but the effect on the video image. For example, I could then partially increase the saturation, the contrast, or whatever else.
In connection with the coupled color corrections, these, again coupled to the alpha change from the movement, could result in freely programmable controls, which I would find mercilessly good...

![Abwedeln](https://user-images.githubusercontent.com/125514250/230779790-d58241ed-f3b4-4101-acdb-aadabd0afee7.png)

What do you think about that?- Would that be technically possible?

PS:
Your controller for 'linked Objects' (green Line with centerpoint and  seesaw, I tried it with two elements) is super.

**Comment by brunoherbelin** [comment]
I like the concept, and we can keep pushing the idea.

Meanwhile, did you try the painting (brush and erasers) in the Texture view?

![Screenshot from 2023-04-11 00-23-48](https://user-images.githubusercontent.com/60393189/231010409-930066c5-a430-4899-83f5-c7405acb5ae3.png)

example above: the video is in the background, and a clone with higher saturation and brightness is in front of it: erasing the clone shows the original behind, and painting the clone is like applying filters with a paint brush. This works for many effects of cloning.

**Comment by flowerhorse** [comment]
> I like the concept, and we can keep pushing the idea.

I think so too.- Maybe an additional function for F3? The distances between the layers are already created there, but they don't play a computational role there at the moment, do they?

> Meanwhile, did you try the painting (brush and erasers) in the Texture view?

Yes, I've tried.- It remembered me on my first Dates with Paintbrush :)
All very rough. There is also still a construction site, I think. I will examine this more closely.
Until now, I have preferred to try out the masks and make my own pictures for filter media. At first glance, Editor was too rigid, too inflexible. But I need to investigate this more closely

**Comment by brunoherbelin** [comment]
While this idea remains to be explored, there is a new feature that could help: choose a source as texture mask for another source: 

![Screenshot from 2023-09-19 19-54-35](https://github.com/brunoherbelin/vimix/assets/60393189/93d7acc4-455d-4cea-b57d-c5a7c764691c)
